models/brain_image_synthesis_latent_diffusion_model/configs/metadata.json

```
@@ -0,0 +1,77 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
```

```suggestion
    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20240725.json",
```
I mentioned it in the other PR on generative models we are merging that it might help to use the current version of the schema. This means you don't need to have an "image" input argument for the network defined here but call it the name that the network actually expects.

---

Sorry, I missed that. Thank you, Eric.